docs/troubleshooting.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: docs
+title: "Troubleshooting"
+permalink: /docs/troubleshooting
+---
+# Troubleshooting
+
+It is important to recognise that things _can_ go wrong. But MicroK8s gives
+you tools to help work out _what_ has gone wrong, as detailed below. Be sure to
+check out the [common issues](#common-issues) section for help resolving the
+most frequently encountered problems.  
+
+### Checking logs
+
+If a pod is not behaving as expected, the first port of call should be the
+logs.
+
+First determine the resource identifier for the pod:
+
+```bash
+microk8s.kubectl get pods
+```
+This will list the currently available pods, for example:
+
+```no-highlight
+NAME                                 READY   STATUS             RESTARTS   AGE
+mk8s-redis-7647889b6d-vjwqm          1/1     Running            0          2m24s
+```
+
+You can then use `kubectl` to view the log. For example, for the simple redis
+pod above:
+
+```bash
+microk8s.kubectl logs mk8s-redis-7647889b6d-vjwqm
+```
+
+### Examining the configuration
+
+If the problem you are experiencing indicates a problem with the configuration
+of the Kubernetes components themselves, it could be helpful to examine the
+arguments used to run these components. These are available in the directory
+`${SNAP_DATA}/args`, which on Ubuntu should point to `var/snap/microk8s/current`.
```

Comment:
It is `/var/snap/microk8s/current`, right? (missing the `/`)

Reply:
well spotted!